lib/rubocop/cop/style/request_referer.rb

```diff
@@ -0,0 +1,38 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for uses of request.referrer.
+      class RequestReferer < Cop
+        include ConfigurableEnforcedStyle
+
+        def on_send(node)
+          if offense?(node)
+            add_offense(node.source_range, node.source_range, message)
+          end
+        end
+
+        def autocorrect(node)
+          ->(corrector) { corrector.replace(node, "request.#{style}") }
+        end
+
+        private
+
+        def offense?(node)
+          receiver = node.receiver
```

Comment:
Why don't you simply decompose the `send` node here? The `receiver.method_name` check on the second line looks pretty odd (and is less efficient).